testdata/dnn/download_models.py

```diff
@@ -781,12 +787,21 @@ def save(self, r):
 #       expected working directory is opencv_extra/testdata/dnn
 if __name__ == '__main__':
     failedSums = []
+    failedGets = []
```

Comment:
I think second list is not needed, we can just rename `failedSums` into `failedModels` and print corresponding message, e.g. "Following models have not been downloaded".